MapboxNavigation/NavigationMapView.swift

```diff
@@ -709,6 +720,190 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     }
 
     // MARK: - Vanishing route line methods
```

Comment:
Would it be possible to move all vanishing route line related methods into another file? `NavigationMapView.swift` becomes pretty heavy.

Reply:
I see. I'll work on this. Thanks for the suggestion.